altair/vegalite/v5/api.py

```diff
@@ -660,6 +665,9 @@ class _Conditional(TypedDict, t.Generic[_C], total=False):
     value: Any
 
 
+IntoCondition: TypeAlias = Union[ConditionLike, _Conditional[Any]]
+
+
```

Comment:
# TODO
Add a docstring, something along the lines of https://github.com/vega/altair/issues/3552#issuecomment-2335128115

Reply:
Fixed in [`2d9f248` (#3567)](https://github.com/vega/altair/pull/3567/commits/2d9f248c04fce106cda742b91748c2746eb6c25c)